src/operator/tensor/matrix_op.cc

```diff
@@ -278,7 +302,10 @@ NNVM_REGISTER_OP(_crop_assign_scalar)
 NNVM_REGISTER_OP(slice_axis)
 .describe(R"code(Slice along a given axis.
 
-Examples:
+Returns a Sliced array along a given `axis` starting from the `begin` index
```

Comment:
"Sliced"  -> "sliced"
Should not be capitalzied. Also shouldn't it be  "returns an array slice along...",  not "returns a sliced array..".